functions.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+require_once 'dbConnect.php';
+
+/**
+ * function to call a var dump with pre tags
+ * @param $variabletoDump - the variable to be dumped
+ */
+function var_dumpPre($variabletoDump) {
+    echo '<pre>';
+    var_dump($variabletoDump);
+    echo '</pre>';
+}
+
+/**
+ * get the collection data from the db
+ * @param $db - the database connection
+ * @return mixed - the data from the database in a assoc array
+ */
+function fetchData($db) {
+    $query = $db->prepare('SELECT `id`, `series`, `introducedYR`, `topSpeedKMH`, `topSpeedMPH`, `withdrawn`, `withdrawnYR`, `imgURL` FROM `shinkansens`;');
+    $query->execute();
+    return $query->fetchAll();
+}
+
+/**
+ * function to build the display box for a collection item
+ * @param array $shinkansen - array for one row from the db
```

Comment:
doesnt have to be from the db, will handle any array with the correct keys. remove mention of the db